JavaScript/JavaScriptSDK.Tests/CheckinTests/Logging.tests.ts

```diff
@@ -202,6 +217,72 @@ class LoggingTests extends TestClass {
                 }
             }
         });
+
+        this.testCase({
+            name: "LoggingTests: Internal events are throttled when the limit is reached",
```

Comment:
This test breaks the [single responsibility principle](http://www.butunclebob.com/ArticleS.UncleBob.PrinciplesOfOod) and verifies implementation of both `throwInternalNonUserActionable` and `throwInternalUserActionable` methods. Making more complex and fragile than necessary. Perhaps testing the `_throttle` method directly and using a spy to verify that `throwInternalNonUserActionable` and `throwInternalUserActionable` invoke it would be cleaner.